modules/redis/redis.c

```diff
@@ -346,6 +346,13 @@ redis_dd_init(LogPipe *s)
   if (!log_dest_driver_init_method(s))
     return FALSE;
 
+  if (!self->key && !self->param1)
```

Comment:
Do we want to exclude all commands without parameter ?
If we are, the error message could be ambiguous, because you could provide a configuration such as
```syslog-ng
destination dd {
   redis( command("DEBUG SEGFAULT"));
};
```
and got the error message.

Reply:
Thanks for the note. I will check some option cases here. 
Probably I will change the if statement.

Reply:
Discussed IRL, redis grammar did not allow to use only key param of command option. At least command option should be used with one command-param also.